### DISCUSSION TITLE
Unable to load mistralai/Mistral-7B-Instruct-v0.2. repository not found

### DISCUSSION THREAD

**Discussion by gningue** [vllm-project/vllm#4414] [Q&A]
Hello,
Did someone have the same problem? I have an error message "repository not found when trying to load the v0.2 model.

`python -m vllm.entrypoints.openai.api_server --model mistralai/Mistral-7B-Instruct-v0.2  --max-model-len 32192 --dtype float16 --gpu-memory-utilization 0.6`
`File "/var/www/vllm/lib/python3.11/site-packages/huggingface_hub/hf_file_system.py", line 789, in _raise_file_not_found
    raise FileNotFoundError(msg) from err
FileNotFoundError: mistralai/Mistral-7B-Instruct-v0.2 (repository not found)`

**Comment by gningue** [comment]
I found a workaround by using a local repository
`python -m vllm.entrypoints.openai.api_server --model /user/.cache/huggingface/hub/models--mistralai--Mistral-7B-Instruct-v0.2/snapshots/41b61a33a2483885c981aa79e0df6b32407ed873  --max-model-len 32192 --dtype float16 --gpu-memory-utilization 0.8`

**Comment by caesar-one** [comment]
Mistral (as well as Llama) requires accepting their terms. I fixed doing:
`$ pip install huggingface_hub` (you can skip this if the package is already installed)
Then:
`$ huggingface-cli login`
Log in using a token from [huggingface.co/settings/tokens](https://huggingface.co/settings/tokens)

**Reply by epignatelli** [reply]
The `vllm` error message should really be improved, then.
